dockers/docker-orchagent/orchagent.sh

```diff
@@ -30,6 +30,8 @@ elif [ "$HWSKU" == "INGRASYS-S8900-54XC" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
 elif [ "$HWSKU" == "INGRASYS-S8900-64XC" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
+elif [ "$HWSKU" == "AS7712-32X" ]; then
```

Comment:
HWSKU follow name convention vendor-MODEL-portstype

In you case, it is Accton-AS7712-32X